### DISCUSSION TITLE
What will cause print speed fade?

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1542] [General Discussion]
I inspected the g code when i sliced it and in a different g code viewer and its showing 30mm/s for all walls and tops for the entire print as well as 50mm/s for infill. For some reason the top 75% of my print is slowing down to less than 10mm/s. The printer still says 100% speed. its not bridging or layer time. its a simple cube.

**Comment by Jonny11bonk** [comment]
Try looking into the Slicer program for the speed differences. The speed that shows on the printer is only a multiplier, so you can speed the whole printing by it.
